projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -71,6 +95,36 @@ export class NavigationUIComponent implements OnDestroy {
     );
   }
 
+  ngOnInit() {
+    if (this.resetMenu) this.resetOnMenuCollapse();
+  }
+
+  resetOnMenuCollapse() {
```

Comment:
Please add jsdocs to these methods to explain what they are doing. Please also add return value annotations (ie. `(): void`).